src/platform/qt/scripting/ScriptingView.cpp

```diff
@@ -62,10 +63,15 @@ void ScriptingView::load() {
 		if (!m_controller->loadFile(filename)) {
 			return;
 		}
+		setMRS(filename);
```

Comment:
You don't need to store this separately, it's always going to be `m_mruFiles.at(0)`--unless `m_mruFiles.isEmpty()` is `true`, in which case there is no most recently loaded script.